source/main.cpp

```diff
@@ -401,7 +406,9 @@ void GameLoop(PlayerInfo &player, TaskQueue &queue, const Conversation &conversa
 			// Events in this frame may have cleared out the menu, in which case
 			// we should draw the game panels instead:
 			(menuPanels.IsEmpty() ? gamePanels : menuPanels).DrawAll();
-			if(isFastForward)
+			if (isPaused)
```

Comment:
style

Reply:
do we want to use present tense like "isFastForward" ?

Reply:
isFastForward is a pre-existing variable name in the code, so I didn't change it.